submission_runner.py

```diff
@@ -426,6 +436,10 @@ def main(_):
                                        FLAGS.tokenizer_vocab_path)
   logging.info('Final %s score: %f', FLAGS.workload, score)
 
+  if FLAGS.wandb and wandb is not None and RANK == 0:
+    wandb.log({"score": score})
```

Comment:
Nit: single quotation marks for consistency.

Reply:
Thank you! Fixed :)